source/class/zx/io/api/server/Request.js

```diff
@@ -59,6 +58,15 @@ qx.Class.define("zx.io.api.server.Request", {
       check: ["callMethod", "subscribe", "poll", "unsubscribe"],
       init: null
     },
+    /**
+     * The raw data received from the transport
+     * Only valid when the request is a REST request
+     */
+    rawData: {
```

Comment:
why are we keeping the raw data, and how is that valid only for certain transport types?